tests/AerospikeCacheTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+declare(strict_types=1);
+
+namespace Lmc\AerospikeCache;
+
+use PHPUnit\Framework\TestCase;
+
+class AerospikeCacheTest extends TestCase
+{
+    /**
+     * @dataProvider doHaveProvider
+     */
+    public function testDoHave(int $statusCode, bool $expectedValue): void
```

Comment:
```suggestion
    public function testShouldConfirmItemPresence(int $statusCode, bool $shouldHaveItem): void
```

Two suggestions here:
1. We tend to use BDD-like names for tests. Their name should explain expected behavior of the tested component - what it *should* do, ie. the test name will always start with "should". From behavior point of view this  component "Should Confirm Item Presence" - and this behavior is covered by the test. See for example [here](https://markhneedham.com/blog/2008/09/04/bdd-style-unit-test-names/) for more about this topic. If the test fails, it is obvious what behavior is no longer working in the tested component, you also see which behaviors are covered by tests and which not.
2. For better clarity I'd use less generic name of the variable (`expectedValue` of what? This can be confusing name.)

Reply:
Changed the name of tests and parameters so they are a bit more clear about their behaviour